packages/config/src/mk/check.mk

```
@@ -1,10 +1,10 @@
-NODE_VERSION?=v$(shell cat .nvmrc)
-NPM_VERSION?=$(shell cat $(NPM_WORKSPACE_ROOT)/package.json | jq -r '.engines.npm')
-
 .PHONY: check/node
+check/node: NPM_VERSION:=$(shell cat $(NPM_WORKSPACE_ROOT)/package.json | jq -r '.engines.npm')
+check/node: NODE_VERSION:=v$(shell head -n1 $(NPM_WORKSPACE_ROOT)/.nvmrc)
 check/node:
 	@node -v | grep $(NODE_VERSION) &> /dev/null || ( \
-		echo "Make sure node-$(NODE_VERSION) is installed (see .nvmrc)"; \
+		echo "Make sure node-$(NODE_VERSION) is installed (please see the root .nvmrc for nvm installation)"; \
+		echo "Once the correct version of node is installed,  re-run your make target"; \
```

super nit, double space detected

```suggestion
		echo "Once the correct version of node is installed, re-run your make target"; \
```

---

🦸 ! 😆